internal/controllers/topology/cluster/patches/engine.go

```diff
@@ -290,7 +290,8 @@ func applyPatchesToRequest(ctx context.Context, req *runtimehooksv1.GeneratePatc
 
 		// If a patch doesn't have a corresponding request item, the patch is invalid.
 		if requestItem == nil {
-			return errors.Errorf("unable to find corresponding request item with uid %q for the patch", patch.UID)
+			log.V(2).Infof("Unable to find corresponding request item with uid %q for the patch", patch.UID)
```

Comment:
Should this be
```suggestion
			log.Infof("Unable to find corresponding request item with uid %q for the patch", patch.UID)
```
Given that it is an error?

Same for the others